test/e2e/predictor/test_autoscaling.py

```diff
@@ -0,0 +1,192 @@
+                limits={"cpu": "100m", "memory": "256Mi"},
+            ),
+        ),
+    )
+
+    annotations = dict()
+    annotations['autoscaling.knative.dev/class'] = 'hpa.autoscaling.knative.dev'
+
+    isvc = V1beta1InferenceService(
+        api_version=constants.KSERVE_V1BETA1,
+        kind=constants.KSERVE_KIND,
+        metadata=client.V1ObjectMeta(
+            name=service_name, namespace=KSERVE_TEST_NAMESPACE,
+            annotations=annotations
+        ),
+        spec=V1beta1InferenceServiceSpec(predictor=predictor),
+    )
+
+    kserve_client = KServeClient(config_file=os.environ.get("KUBECONFIG", "~/.kube/config"))
+    kserve_client.create(isvc)
+    kserve_client.wait_isvc_ready(service_name, namespace=KSERVE_TEST_NAMESPACE)
+    pods = kserve_client.core_api.list_namespaced_pod(KSERVE_TEST_NAMESPACE,
+                                                      label_selector='serving.kserve.io/inferenceservice={}'
+                                                      .format(service_name))
+
+    isvc_annotations = pods.items[0].metadata.annotations
+
+    assert(isvc_annotations['autoscaling.knative.dev/metric'] == 'cpu')
+    assert(isvc_annotations['autoscaling.knative.dev/target'] == '50')
+    res = predict(service_name, "./data/iris_input.json")
+    assert res["predictions"] == [1, 1]
+    kserve_client.delete(service_name, KSERVE_TEST_NAMESPACE)
+
+
+def test_sklearn_kserve_raw():
+    service_name = "isvc-sklearn-scale-raw"
+    predictor = V1beta1PredictorSpec(
+        min_replicas=1,
+        scale_metric='cpu',
+        scale_target=50,
+        sklearn=V1beta1SKLearnSpec(
+            storage_uri="gs://kfserving-examples/models/sklearn/1.0/model",
+            resources=V1ResourceRequirements(
+                requests={"cpu": "100m", "memory": "256Mi"},
+                limits={"cpu": "100m", "memory": "256Mi"},
+            ),
+        ),
+    )
+
+    annotations = dict()
+    annotations['serving.kserve.io/deploymentMode'] = 'RawDeployment'
+
+    isvc = V1beta1InferenceService(
+        api_version=constants.KSERVE_V1BETA1,
+        kind=constants.KSERVE_KIND,
+        metadata=client.V1ObjectMeta(
+            name=service_name, namespace=KSERVE_TEST_NAMESPACE,
+            annotations=annotations
+        ),
+        spec=V1beta1InferenceServiceSpec(predictor=predictor),
+    )
+
+    kserve_client = KServeClient(config_file=os.environ.get("KUBECONFIG", "~/.kube/config"))
+    kserve_client.create(isvc)
+    kserve_client.wait_isvc_ready(service_name, namespace=KSERVE_TEST_NAMESPACE)
+    api_instance = kserve_client.api_instance
+    hparesp = api_instance.list_namespaced_custom_object(group='autoscaling', version='v1',
+                                                         namespace=KSERVE_TEST_NAMESPACE,
+                                                         plural='horizontalpodautoscalers')
+
+    assert(hparesp['items'][0]['spec']['targetCPUUtilizationPercentage'] == 50)
```

Comment:
FAILED predictor/test_autoscaling.py::test_sklearn_kserve_raw - assert 80 == 50

Seems like it is getting the default value 80 instead of what's specified on inference service

Reply:
Yes, its always passing on my local, tested with others as well, not, its working for them as well in local, but failing in argo...